src/Storages/MergeTree/MergeTreeDataPartChecksum.h

```diff
@@ -36,6 +38,16 @@ struct MergeTreeDataPartChecksum
     void checkSize(const IDataPartStorage & storage, const String & name) const;
 };
 
+inline void writeBinary(const MergeTreeDataPartChecksum::uint128 & x, WriteBuffer & buf)
+{
+    writePODBinary(x, buf);
+}
+
+inline void readBinary(MergeTreeDataPartChecksum::uint128 & x, ReadBuffer & buf)
+{
+    readPODBinary(x, buf);
+}
+
```

Comment:
Since this data structure reoccurs frequently in other parts of the code as well, perhaps you could move these into `IO/ReadHelpers.h` and `IO/WriteHelpers.h`, respectively.